src/repository.ts

```diff
@@ -709,23 +709,25 @@ export class JJRepository {
             ret.change.isConflict = value === "true";
             break;
           case "diff.summary()": {
-            const changeRegex = /^(A|M|D|R) (.+)$/;
+            const changeRegex = /^(A|M|D|R|C) (.+)$/;
```

Comment:
I don't know if all of these have UI support (never heard of IntentToAdd), but here's the list of possible states:
https://github.com/jj-vcs/jj/blob/9e5205daee5a551598c9bace4becd43c7d372b3c/lib/testutils/src/git.rs#L234-L245

Reply:
I'm hoping we'll be good now based on https://github.com/jj-vcs/jj/blob/9e5205daee5a551598c9bace4becd43c7d372b3c/cli/src/diff_util.rs#L1905